### PR TITLE
Help Tickets: Call /helpticket when /report is supplied with a target

### PR DESCRIPTION
**Pull request by Hawkiee**
http://www.smogon.com/forums/threads/suggestions.3534365/page-66#post-7652306

"Can there be an auto-reply thing to /report? People often put /report [name] and all it tells them is that the command does not exist. Can it be changed to where it says something that specifies all they need to do is type /report and not include a name. Or maybe even make it possible to report with a name included to the command. A lot of people face this issue in the Help room i noticed."

### CODE REVIEW

File: chat-plugins/helptickets.js [hunk 1]
```diff
@@ -446,7 +446,9 @@ exports.pages = {
 exports.commands = {
 	requesthelp: 'helpticket',
 	helprequest: 'helpticket',
-	report: 'helpticket',
+	report: function (target, room, user, connection) {
+		return this.parse('/helpticket');
```
Comment (KrisOnline) [inline chat-plugins/helptickets.js]: this doesnt do what you want it to do; this just means that doing `/report` will do what it already does

Comment (KrisOnline) [reply chat-plugins/helptickets.js]: and seeing as the point of the pull request is to have `/report` open the Ticket window, this is what it already does, which makes this PR useless?

Comment (Hawkiee) [reply chat-plugins/helptickets.js]: i tested it on localhost and '/report hawkie' opened the view-help-request room like it should, currently the server just returns 'The command "/report hawkie" does not exist..' etc

Comment (Hawkiee) [reply chat-plugins/helptickets.js]: The ticket window currently doesn't open if you do `/report target`, which a lot of people do as they're using the command to report a user. With this change `/report` ignores the target and just opens the ticket window.

Comment (KrisOnline) [reply chat-plugins/helptickets.js]: ok but what about the rest of the aliases?

Comment (Hawkiee) [reply chat-plugins/helptickets.js]: the other aliases were left alone so `/ht ban/unban` etc are unaffected. `/report` is the only alias that causes users problems accessing the ticket window.  It does mean staff won't be able to use `/report [command]` though.

File: chat-plugins/helptickets.js [hunk 2]
```diff
@@ -446,7 +446,17 @@ exports.pages = {
 exports.commands = {
 	requesthelp: 'helpticket',
 	helprequest: 'helpticket',
-	report: 'helpticket',
+	report: function (target, room, user, connection) {
+		let message = toId(target);
+		if (message.includes('list') ||
```
Comment (KrisOnline) [inline chat-plugins/helptickets.js]: I don’t see why these need to be on separate lines either

Comment (Hawkiee) [reply chat-plugins/helptickets.js]: you're right i started it that way and didn't realise there were so few commands lol

File: chat-plugins/helptickets.js [hunk 3]
```diff
@@ -446,7 +446,13 @@ exports.pages = {
...
 	requesthelp: 'helpticket',
 	helprequest: 'helpticket',
-	report: 'helpticket',
+	report: function (target, room, user, connection) {
+		let message = toId(target);
+		if (message.includes('list') || message.includes('escalate') || message.includes('close') || message.includes('ban') || message.includes('delete')) {
+			return this.parse('/helpticket ' + target);
+		}
+		return this.parse('/helpticket');
+	},
```
Comment (Zarel) [inline chat-plugins/helptickets.js]: Can you move this to a different section?

Comment (Hawkiee) [reply chat-plugins/helptickets.js]: it's below the helpticket object now where i assume it should go

**Pull request closed without merging**